src/System.Security.Cryptography.Cng/tests/ECDsaCngTests.cs

```diff
@@ -2,98 +2,88 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.IO;
-using System.Linq;
-using System.Text;
-using System.Collections.Generic;
-using System.Security.Cryptography;
-
+using System.Security.Cryptography.EcDsa.Tests;
 using Test.Cryptography;
 using Xunit;
 
 namespace System.Security.Cryptography.Cng.Tests
 {
-    public static class EcDSACngTests
+    public class ECDsaCngTests : ECDsaTestsBase
     {
         [Fact]
-        public static void TestPositiveVerify256()
+        public void TestPositive256WithBlob()
```

Comment:
But everything in ECDsaTestsBase seems to be static helpers (no instance methods, no Fact or Theory attributes).  So everything here should still be able to be static methods, and then the xunit framework won't need to instantiate the test classes.

It's not really a big deal, though.